testsuite/tests/lib-stdlib/int_of_string.ml

```diff
@@ -0,0 +1,69 @@
+(* TEST
```

Comment:
That doesn't seem to be good place for these tests. What would be a better place?